src/glfs-cli.c

```diff
@@ -271,7 +271,7 @@ parse_options (struct cli_context *ctx)
         opterr = 0;
 
         while (true) {
-                opt = getopt_long (argc, argv, "o:", long_options,
```

Comment:
The `'x'` used in the case statement is just used to refer to the long option. If it's not included in the optstring, then `getopt_long` doesn't expose it as a short option at runtime.

```c
static struct option const long_options[] =
{
        {"help", no_argument, NULL, 'x'},
        {NULL, no_argument, NULL, 0}
}

opt = getopt_long (argc, argv, "", long_options, &option_index);

switch (opt) {
  case 'x': //
}
```

vs.

```c
static struct option const long_options[] =
{
        {"help", no_argument, NULL, 'x'},
        {NULL, no_argument, NULL, 0}
}

opt = getopt_long (argc, argv, "x", long_options, &option_index);

switch (opt) {
  case 'x': //
}
```

In the former calling `./bin/gf -x` results in an error whereas the latter does not.